index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
```

Comment:
Great start @sadat-25 but I think you have followed the example to closely. Change it up, use your own ideas.

Please make your page more distinct. Different colors, fonts, and borders please.

<img width="450" alt="sadat_homework_001" src="https://user-images.githubusercontent.com/13404719/78423591-9747de00-7667-11ea-97a0-137895d754df.png">

Reply:
okay i will change it